SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityAgentReconciliationPoller.java

```diff
@@ -55,7 +55,7 @@
   @Override
   public void runActionOnPoll() {
     refereshSlavesAndRacks();
-    checkDeadSlaves();
+    checkInactiveSlaves();
```

Comment:
totally missed this in earlier PRs. Can you do a slaves -> agents rename as you go through this PR please. Really thought I had caught most of those already

Reply:
May do more of this in a separate PR; upon renaming tests, the order that the tests run changes and causes some more test failures likely due to interference between tests